Dockerfile

```diff
@@ -45,8 +45,15 @@ ENV JAVA_HOME /usr/lib/jvm/java-8-oracle
 # Install python 2.7 and dependancies for Genetorrent.
 #
 
-RUN apt-get install -y python
- 
+RUN apt-get install -y python && \
+    mkdir /icgc/cli/clients
+COPY /lib/* /icgc/
+COPY /lib/cli/* /icgc/cli/
```

Comment:
Maybe its best to just use one  `COPY` and a `.dockerignore`. Lets future maintenance.